libs/preprocessor/tests/warnings/pw3_padded_arg_inner/source.hpp

```diff
@@ -0,0 +1,6 @@
+#define HELLO(var1, var2) INNER(Hello, var1##var2)
+#define GREET(var1, var2) INNER(Greetings, var1##var2)
```

Comment:
```suggestion
#define GREET(var1, var2) HELLO(Greetings, var1##var2)
```
Is that what was intended here? Otherwise I don't understand the difference